test/dlc_tests/sanity/test_ecr_scan.py

```diff
@@ -143,6 +147,8 @@ def helper_function_for_leftover_vulnerabilities_from_enhanced_scanning(
 
     :param image: str Image URI for image to be tested
     :param python_version: str, This parameter is used for extracting allowlist for canary image uris that do not have a python version in it.
+    :param minimum_sev_threshold: str, If minimum_sev_threshold is set vulnerabilities with severity < minimum_sev_threshold will not be taken into consideration.
+    :param allowlist_removal_enabled: boolean, Value of this parameter decides if we should remove allowlisted vulnearbilities from the scanner results.
```

Comment:
Please add docstring for `remove_non_patchable_vulns`

Reply:
Added the docstring. Thanks!!